src/app/stake/btc/page.tsx

```diff
@@ -126,87 +121,146 @@ const StakeBTCPage = () => {
   return (
     <>
       <div className="lg:w-1/2 mx-auto px-4 md:px-16 lg:px-0">
-        {btcWallet ? (
-          <div className="flex space-x-4 mb-4">
-            <div className="flex-1 p-4 bg-gray-200">
-              <div className="text-sm font-medium text-gray-600">TOTAL BTC</div>
-              <div className="text-2xl ">
-                {satoshiToBtc(btcWalletBalanceSat)}
+        <div className="flex items-baseline mb-4">
+          <h1 className="text-[#332B29] text-2xl font-bold mr-[16px]">
```

Comment:
we should using custom values like this. brand colors should be added to tailwinds config. we should have a sensible naming convention for them like surface, secondary, primary, etc.